src/MorganStanley.ComposeUI.Host/Modules/CommunicationModule.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+* Morgan Stanley makes this available to you under the Apache License,
+* Version 2.0 (the "License"). You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0.
+*
+* See the NOTICE file distributed with this work for additional information
+* regarding copyright ownership. Unless required by applicable law or agreed
+* to in writing, software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+* or implied. See the License for the specific language governing permissions
+* and limitations under the License.
+*/
+
+using MorganStanley.ComposeUI.Interfaces;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.IO;
+using ComposeUI.Messaging.Client;
+
+namespace MorganStanley.ComposeUI.Host.Modules;
+
+internal class CommunicationModule : IModule, IDisposable
+{
+    Process? _process;
+    private bool _disposed;
+
+    public async Task Initialize(IMessageRouter messageRouter)
+    {
+        _process = new Process();
+        _process.StartInfo.UseShellExecute = false;
+        //_process.StartInfo.CreateNoWindow = true;
+
+        var location = Path.GetFullPath("..\\..\\..\\ComposeUI.Messaging.Server\\bin\\Debug\\ComposeUI.Messaging.Server.exe");
```

Comment:
This could point to the publish output directory instead. Or make it a command line argument.

Reply:
I would keep it this way for this stage, as it works "out of the box" for the developer.
I think we should discuss how to arrange the output of the different pieces and how to do a better integration for the next step.

Reply:
LGTM for this stage. I saw it working.